docs/usage.md

```diff
@@ -69,119 +115,312 @@ cat ./compilation.json
+    --logfile compile_commands.json
 ```
 For more details regarding the skip file format see
 the [user guide](analyzer/user_guide.md#skip).
 
-c) **Select source files from the compilation database**
+#### Analyze sources what explicitly seleced from the compilation database<a name="analize-explicit-files"></a>
 You can select which files you would like to analyze from the compilation
 database. This is similar to the skip list feature but can be easier to quickly
 analyze only a few files not the whole compilation database.
 
-Let's assume you have the compilation database in `compile_commands.json`.
+Undo filesystem modifications as described in the
+[previous section](#narrow-files). Re-do analyze and perform automatic fix. The
+following command re-analyzes only the `main.cpp` file.
 
 ```sh
-CodeChecker check ./compile_commands.json -o reports --file "*cmd-find.c"
+CodeChecker check -output ./reports --file "*/src/main.cpp" \
+    --enable sensitive --logfile ./compile_commands.json
 ```
 
 Absolute directory paths should start with `/`, relative directory paths should
 start with `*` and it can contain path glob pattern. Example:
 `/path/to/main.cpp`, `lib/*.cpp`, `*/test*`.
 
 
-### Analysis Failures <a name="analysis-failures"></a>
+## Step 5: Store analysis results in a CodeChecker DB and visualize results<a name="step-5"></a>
+You can store the analysis results in a central database and view the results
+in a web viewer:
+
+1. Start the CodeChecker server locally on port 8555 (using SQLite DB, which is
+not recommended for multi-user central deployment) create a workspace
+directory, where the database will be stored.
+
+```sh
+cd <repo root dir>
+mkdir ./ws
+CodeChecker server --workspace ./ws -v 8555
+```
+A default product called `Default` will be automatically created where you can
+store your results.
+
+2. Store the results in the server under name "example" (in the `Default`
+product):
+
+```
+CodeChecker store ./reports --name example --url http://localhost:8555/Default
+```
+
+The URL is in `PRODUCT_URL` format:
+`[http[s]://]host:port/ProductEndpoint`
+Please note that if you start the server in secure mode (with SSL) you will
+need to use the `https` protocol prefix. The default protocol is `http`.
+See [user guide](web/user_guide.md#product_url-format) for detailed
+description of the `PRODUCT_URL` format.
+
+3. View the results in your web browser
+firefox `http://localhost:8555/Default` &
+
+### Definition of "run"<a name="run-definition"></a>
+When you develop your project in discrete steps in time, you can analyze and
+store the current snapshot of your project. During store, the locally generated
+"database"/"analysis report" are uploaded to the CodeChecker server. The store
+command has a `--name` opton that "gives name"/"identifies" the run. Subsequent
+store actions with the same name make a run. The ordered list in time of store
+actions constitute the "run history".
```

Comment:
A run, in the simplest case is a single analysis snapshot of your software.

You can follow up the quality status of your product by storing the analysis results of consecutive git commits of a git branch into the same run. In this case, you will be able to follow-up the outstanding results and fixed reports in the statistics views (in the web ui). 
So a run represents the analysis status of a single branch of your code, with a given analysis configuration. You can also record the source code version associated with the analysis with the `--tag`.

Make sure, that you use the analysis configuration when updating a run, because a changed analysis configuration can make new reports to appear or reports to disappear.

Reply:
Done.